src/tyr/route_serializer_osrm.cc

```diff
@@ -682,13 +648,35 @@ valhalla::baldr::json::MapPtr serializeIncident(const TripLeg_Node_Incident& inc
   if (incident.type()) {
     metadata_json->emplace("incident_type", valhalla::incidentTypeToString(incident.type()));
```

Comment:
Can we just rename this to `type` - it's already in the `incidents[]` array, this field doesn't need to be prefixed with `incident_`